src/EditorFeatures/CSharpTest/Diagnostics/UpgradeProject/UpgradeProjectTests.cs

```diff
@@ -69,7 +69,7 @@ void A()
 }",
                 LanguageVersion.CSharp7,
                 new CSharpParseOptions(LanguageVersion.CSharp6),
-                index: 1);
+                index: 0);
```

Comment:
Perhaps remove the `index: 0` parameter in all the modified tests (since no longer useful).